content/docs/experts/map.toml

```diff
@@ -65,7 +65,7 @@ familiar = []
 name = "Parser"
 directories = []
```

Comment:
@Centril can you fill the directories involved here and maybe more if you know? :)

Reply:
Well it's largely just https://github.com/rust-lang/rust/tree/master/src/libsyntax/parse, https://github.com/rust-lang/rust/tree/master/src/libsyntax/util, and https://github.com/rust-lang/rust/blob/master/src/libsyntax/ast.rs